modules/fundamental/src/sbom/model/details.rs

```diff
@@ -255,11 +259,31 @@ impl SbomAdvisory {
 
             let mut packages = vec![];
             if let Some(package) = &product.product_status.package {
-                let package = SbomPackage {
-                    name: package.to_string(),
-                    ..Default::default()
+                // We need to enable search on namespace/name pattern as well
+                // Also, from the brief look at it, the search is case sensitive
+                // which doesn't cover all use cases
+                let parts = package.split('/').collect::<Vec<_>>();
```

Comment:
To my understanding the "package" here is the CSAF (advisory) internal package name. Taking that and looking up through a PURL "by name" with a sub-string match.

I think this is highly Red Hat specific and error prone.

And, as we seem to this this in a for loop, this might have a performance impact that we should check.